examples/ex1.cpp

```diff
@@ -168,7 +177,10 @@ int main(int argc, char *argv[])
    //    domain integrator.
    BilinearForm *a = new BilinearForm(fespace);
    if (pa) { a->SetAssemblyLevel(AssemblyLevel::PARTIAL); }
-   a->AddDomainIntegrator(new DiffusionIntegrator(one));
+   if (ea) { a->SetAssemblyLevel(AssemblyLevel::ELEMENT); }
+   if (fa) { a->SetAssemblyLevel(AssemblyLevel::FULL); }
+   // a->AddDomainIntegrator(new DiffusionIntegrator(one));
+   a->AddDomainIntegrator(new MassIntegrator(one));
```

Comment:
Look like these changes where just for testing? perhaps move to unit test?

Reply:
The change in `ex1` are going to be discarded, I left them temporarily for testing purpose for the reviewers We have to decide how we want to to show this new feature.